.github/workflows/test.yml

```diff
@@ -12,51 +16,44 @@ on:
       - develop
 
 jobs:
-  # wpa:
-  #   runs-on: ubuntu-latest
-
-  #   strategy:
-  #     matrix:
-  #       snapshot: [
-  #         "3593222a181f0f91824947dbd06985d5", # WP 4.9
-  #         "9ea863cebfcbb416c6334a6bef6552f0", # WP 4.9 & Gutenberg 4.9
-  #         "803989004f4b101a644ebfe9bb67e712", # WP 5.4
-  #       ]
-
-  #   steps:
-  #   - name: Checkout
-  #     uses: actions/checkout@v2.4.0
-
-  #   - name: Set PHP version
-  #     uses: shivammathur/setup-php@v2
-  #     with:
-  #       php-version: '7.2'
-  #       coverage: none
-
-  #   - name: Install dependencies
-  #     run: composer install --ignore-platform-reqs
-
-  #   - name: Setup WP Snapshots
-  #     run: ./vendor/bin/wpsnapshots configure 10up --aws_key=${{ secrets.AWS_ACCESS_KEY_ID }} --aws_secret=${{ secrets.AWS_SECRET_ACCESS_KEY }} --user_name=wpacceptance --user_email=wpacceptance@10up.com
-
-  #   - name: Test
-  #     run: ./vendor/bin/wpacceptance run --snapshot_id=${{ matrix.snapshot }}
 
   phpunit:
-    runs-on: ubuntu-latest
+    name: ${{ matrix.php }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # We claim to support from 5.6+ but WP Mock only supports 7.1+
+        # Also an issue with WP Mock not working on PHPUnit 9.5+, which PHP 7.3+ needs.
```

Comment:
Testing locally on PHP 7.4 I managed to resolve this by adding `'phpunit/phpunit':'>=7.0 <9.0'` to this projects composer file.

Further bumping to PHP 8.0 still threw some errors, `Mockery\Exception\NoMatchingExpectationException`, but two more versions of PHP seems better than nothing.

Reply:
Ah, nice catch. Adding that brought 7.3 and 7.4 and I was able to fix the couple broken tests so we also have 8.0 and 8.1 now. Thanks!